docs/src/adoc/index.adoc

```diff
@@ -4636,7 +4636,14 @@ the `User.id` and `User.name` fields. Likewise for `r_id` and `r_name` into the
 The link:{jdbidocs}/sqlobject/config/RegisterConstructorMapper.html[@RegisterConstructorMapper^] annotation may be repeated multiple times on
 the same type or method to register multiple constructor mappers.
 
+==== @RegisterCollectorFactory
```

Comment:
RegisterCollector ?

Reply:
I think this would be a better fit after `@UseRowReducer`. The docs are so bare bones that, without an example, this will probably not find any users.